### ISSUE TITLE
Data not binding to choropleth in latest folium 0.2.0.dev (but worked in 0.1.6)

### ISSUE THREAD

**Issue by alexandercbooth** [python-visualization/folium#316]
Hi @ocefpaf & @BibMartin et al., 

First, thank you all for making such a wonderful tool!

I recently updated from the folium version currently on pip (0.1.6) to the current master (0.2.0.dev). A choropleth bound with data in 0.1.6 is no longer rendering in 0.2.0.dev. My code and images are displayed below. 

# In folium 0.1.6
```
zip_geo = r'newzip4.json'
m = folium.Map(location=[45,-72], zoom_start=4, tiles='cartodbpositron')
m.geo_json(geo_path=zip_geo, data_out='data4.json', data=df,
             columns=['ZIP3', 'count'],
             key_on='feature.properties.ZIP3',
             fill_color='YlGnBu', fill_opacity=0.95, line_opacity=0.9, 
             legend_name='Count by Location', topojson='objects.newzip3_4326', threshold_scale = [1,2500,5000,7500, 12000])
m
```

### This produces the following map:

![screen shot 2016-01-04 at 12 20 16 pm](https://cloud.githubusercontent.com/assets/15242567/12095590/9f71df74-b2dd-11e5-8594-dbbeacc0c20a.png)

# In folium 0.2.0.dev
```
zip_geo = r'newzip4.json'
m = folium.Map(location=[45,-72], zoom_start=4, tiles='cartodbpositron')
m.choropleth(geo_path=zip_geo, data_out='data4.json', data=df,
             columns=['ZIP3', 'count'],
             key_on='feature.properties.ZIP3',
             fill_color='YlGnBu', fill_opacity=0.95, line_opacity=0.9, 
             legend_name='Count by Location', topojson='objects.newzip3_4326', threshold_scale = [1,2500,5000,7500, 12000])
m
```

### This produces the same map, but without the data bound to it:

![screen shot 2016-01-04 at 12 22 25 pm](https://cloud.githubusercontent.com/assets/15242567/12095623/e048e358-b2dd-11e5-97d9-5ce183121696.png)
 
I apologize if this is just due to my misunderstanding of binding data to choropleths in the latest version. Any help would be greatly appreciated. 

Thanks so much!

Alex

**Comment by ocefpaf** [comment]
> I apologize if this is just due to my misunderstanding of binding data to choropleths in the latest version. 

It seems OK to me.  Can you share your data so we can investigate this further?

**Comment by alexandercbooth** [comment]
Hi @ocefpaf,

Thanks very much for looking into this for me. Sure, I will attach the topojson zip code file and the data (which is just fabricated for testing) as a csv in a zip file. The topojson file is the 3 digit us zip code tabulation area that I converted from a shape file. 

Thanks!
[foldata.zip](https://github.com/python-visualization/folium/files/77541/foldata.zip)

**Comment by BibMartin** [comment]
Hi @alexandercbooth 
Thanks for issuing that point.

You're true, the binding has disapeared for TopoJson since #266 and it's my fault.
I'll try to tackle this soon, unless you feel like making a PR.

**Comment by alexandercbooth** [comment]
Thanks so much Martin!! I very much appreciate you and @ocefpaf looking into it and finding a fix.

**Comment by ocefpaf** [comment]
> I very much appreciate you and @ocefpaf looking into it and finding a fix.

Well... @BibMartin did all the work  :wink:

And he already has a fix #317.

Thanks @BibMartin!

**Comment by alexandercbooth** [comment]
Haha, well, appreciated all the same. Works great, thanks!
